lib/src/WebSocketConnectionImpl.cc

```diff
@@ -165,6 +167,15 @@ void WebSocketConnectionImpl::send(const std::string_view msg,
     send(msg.data(), msg.length(), type);
 }
 
+void WebSocketConnectionImpl::send(Json::Value &json,
+                                   const WebSocketMessageType type)
+{
+    Json::FastWriter writer;
```

Comment:
Some configuration options may have an impact on json strings, please refer to the implementation in HttpResponse:
https://github.com/drogonframework/drogon/blob/4cbac301fbebb369dba8935f4f727a1ba5b9c513/lib/src/HttpResponseImpl.cc#L114-L129